packages/webdriverio/webdriverio.d.ts

```diff
@@ -102,6 +108,8 @@ declare namespace WebdriverIOAsync {
 declare var browser: WebdriverIOAsync.BrowserObject;
 declare var $: $;
 declare var $$: $$;
+declare var react$: react$;
+declare var react$$: react$$;
```

Comment:
These are not declared as global variables, it is only for `$` and `$$`

Reply:
@christian-bromann Fixed, thanks

Reply:
ah, tests failed
Is that true that above are not necessary?